code/clue.js

```diff
@@ -1,59 +1,317 @@
+  weight: 45,
+  length: '50 cm',
+  color: 'LightSteelBlue',
+  image: 'assets/weapon/axe.png'
+};
+
+const bat = {
+  name: 'Bat',
+  weight: 25,
+  length: '55 cm',
+  color: 'AntiqueWhite',
+  image: 'assets/weapon/bat.png'
+};
+
+const trophy = {
+  name: 'Trophy',
+  weight: 35,
+  length: '45 cm',
+  color: 'LightYellow',
+  image: 'assets/weapon/trophy.png'
+};
+
+const pistol = {
+  name: 'Pistol',
+  weight: 25,
+  length: '35 cm',
+  color: 'Thistle',
+  image: 'assets/weapon/pistol.png'
+};
+
+// Objects for all rooms
+const diningRoom = {
+  name: 'Dining Room',
+  color: 'PapayaWhip',
+  image: 'assets/room/dining.png'
+};
+
+const kitchen = {
+  name: 'Kitchen',
+  color: 'SeaShell',
+  image: 'assets/room/kitchen.png'
+};
+
+const study = {
+  name: 'Study',
+  color: 'PapayaWhip',
+  image: 'assets/room/study.png'
+};
+
+const library = {
+  name: 'Library',
+  color: 'SeaShell',
+  image: 'assets/room/library.png'
+};
 
-// THE ROOMS ONLY HAS A NAME SO NO NEED FOR OBJECTS THERE.
+const billiardRoom = {
+  name: 'Billiard Room',
+  color: 'AntiqueWhite',
+  image: 'assets/room/billiard.png'
+};
 
-// NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
+const lounge = {
+  name: 'Lounge',
+  color: 'PapayaWhip',
+  image: 'assets/room/lounge.png'
+};
 
+const ballroom = {
+  name: 'Ballroom',
+  color: 'AntiqueWhite',
+  image: 'assets/room/ballroom.png'
+};
+
+const spa = {
+  name: 'Spa',
+  color: 'SeaShell',
+  image: 'assets/room/spa.png'
+};
+
+const livingRoom = {
+  name: 'Living Room',
+  color: 'AntiqueWhite',
+  image: 'assets/room/livingroom.png'
+};
+
+const theater = {
+  name: 'Theater',
+  color: 'PapayaWhip',
+  image: 'assets/room/theater.png'
+};
+
+const garden = {
+  name: 'Garden',
+  color: 'PapayaWhip',
+  image: 'assets/room/garden.png'
+};
+
+
+
+// ARRAYS FOR SUSPECTS, WEAPONS AND ROOMS //
+
+//Array for suspects
 const suspects = [
   mrGreen,
+  colMustard,
+  mrsPeacock,
+  profPlum,
+  missScarlet,
   mrsWhite
-  // ...  and the rest
 ]
 
-const weapons = []
+//Array for weapons
+const weapons = [
+  rope,
+  knife,
+  candlestick,
+  dumbbell,
+  poison,
+  axe,
+  bat,
+  trophy,
+  pistol
+];
 
-const rooms = []
+// Array for rooms
+const rooms = [
+  diningRoom,
+  kitchen,
+  study,
+  library,
+  billiardRoom,
+  lounge,
+  ballroom,
+  spa,
+  livingRoom,
+  theater,
+  garden
+];
 
-// THIS FUNCTION WILL RANDOMLY SELECT ONE ITEM FROM THE ARRAY THAT YOU PASS IN TO THE FUNCTION.
-// YOU DON'T NEED TO CHANGE THIS, JUST TRY TO UNDERSTAND IT. AND HOW TO USE IT.
+//FUNCTION FOR RANDOMLY SELECTING ONE ITEM FROM THE ARRAY THAT'S PASSED INTO THE FUNCTIONS FOR SUSPECTS, WEAPONS AND ROOMS.
 const randomSelector = array => {
   return array[Math.floor(Math.random() * array.length)]
 }
 
-// CREATE AN OBJECT THAT KEEPS THE MYSTERY.
-// With a killer, a weapon and a room.
-// The values will be set later.
+//OBJECT THAT KEEPS THE MYSTERY //
+const mystery = {
+ killer: null,
+ weapon: null,
```

Comment:
Very good initialisation of mystery object - `null` values are the most proper :)